config/payment.php

```diff
@@ -9,7 +9,7 @@
     | You can switch to a different driver at runtime.
     |
     */
-    'default' => 'zarinpal',
+    'default' => 'behpardakht',
```

Comment:
please don't change the default gateway (driver).

most people use `Zarinpal`, so default gateway must be `Zarinpal`.